RoddyCore/src/de/dkfz/roddy/config/converters/BashConverter.groovy

```diff
@@ -245,12 +244,20 @@ class BashConverter extends ConfigurationConverter {
             return '"' + value + '"'
     }
 
-    static String convertListToBashArrayString(List list) {
-        "(${list.collect { it.toString() }.join(" ")})" as String
+    static String convertListToBashArrayString(List list, doQuote = true) {
+        String result = "(${list.collect { it.toString() }.join(" ")})"
+        if (doQuote)
```

Comment:
``` lang=java
if (doQuote)
  return "\"${result}\""  ?
return result
```

Reply:
Think this is only a minor gain over 

```groovy
if (cond)
  "a"
else
  "b"
```

One could also argue that from a Groovy perspective the current code is more idiomatic, as it does not use unnecessary `return` statements.